integration-tests/src/runtime_tests.rs

```diff
@@ -0,0 +1,59 @@
+//! Test that the runtime is config is good and secured, no sending XCM
```

Comment:
Most Substrate integration test suits use `subxt` or `polkadotjs/api` but I guess these tests are all about running a single node that interacts with built-in runtimes, right?

If so, then it is not a problem IMO but it might be difficult to interact with third-party runtimes like Moonbeam or Karura in the future. If not, then I am speaking gibberish

Reply:
akala and hydradx are simlurated like here. we will integrate these first, but  @andor0 work on subxt stuff